examples/get-started/pure-js-google-maps/app.js

```diff
@@ -0,0 +1,49 @@
+/* global document, google */
+import {GoogleMapsOverlay as DeckOverlay} from '@deck.gl/google-maps';
+import {GeoJsonLayer} from '@deck.gl/layers';
+
+// source: Natural Earth http://www.naturalearthdata.com/ via geojson.xyz
+const GEOJSON =
+  'https://d2ad6b4ur7yvpq.cloudfront.net/naturalearth-3.3.0/ne_110m_admin_1_states_provinces_shp.geojson'; //eslint-disable-line
+
+// Retrieving GOOGLE_MAPS_API_KEY from the environment variable
+const gmUrl = `https://maps.googleapis.com/maps/api/js?key=${
+  process.env.GOOGLE_MAPS_API_KEY // eslint-disable-line
+}&libraries=visualization&v=3.34`;
+
+function loadScript(url) {
```

Comment:
Assume you considered putting the script injection helper into `@deck.gl/google-maps` module and decided against it?

Reply:
We do not inject Mapbox script in the `@deck.gl/mapbox` module. The primary audience of these modules are existing developers of Mapbox/Google Maps who already import the dependencies with their own setup, either dynamically or with a script tag.